firewalldb/actions_test.go

```diff
@@ -171,7 +174,7 @@ func TestListActions(t *testing.T) {
 		actionIds++
 
 		actionReq := &AddActionReq{
-			SessionID:          sessionID,
+			MacaroonIdentifier: sessionID,
```

Comment:
should we set `SessionID` here as well?

Reply:
want to keep the tests as close to how they are today as possible - then will expand the tests to cover more cases once the SQL impl is added 👍